src/radio/radio.js

```diff
@@ -117,51 +117,54 @@ class Radio extends React.Component<RadioPropsT, RadioStateT> {
       $value: this.props.value,
     };
 
-    const label = (
-      <Label {...sharedProps} {...labelProps}>
-        {this.props.children}
-      </Label>
-    );
+    // const label = (
+    //   <Label {...sharedProps} {...labelProps}>
+    //     {this.props.children}
+    //   </Label>
+    // );
 
     return (
       <React.Fragment>
         <Root
-          data-baseweb="radio"
-          onMouseEnter={this.onMouseEnter}
-          onMouseLeave={this.onMouseLeave}
-          onMouseDown={this.onMouseDown}
-          onMouseUp={this.onMouseUp}
-          {...sharedProps}
-          {...rootProps}
+        // data-baseweb="radio"
+        // onMouseEnter={this.onMouseEnter}
+        // onMouseLeave={this.onMouseLeave}
+        // onMouseDown={this.onMouseDown}
+        // onMouseUp={this.onMouseUp}
+        // {...sharedProps}
+        // {...rootProps}
         >
-          {isLabelTopLeft(this.props.labelPlacement) && label}
+          {/* {isLabelTopLeft(this.props.labelPlacement) && label} */}
+
           <RadioMarkOuter {...sharedProps} {...radioMarkOuterProps}>
             <RadioMarkInner {...sharedProps} {...radioMarkInnerProps} />
           </RadioMarkOuter>
+
           <Input
-            aria-invalid={this.props.error || this.props.isError || null}
+            // aria-invalid={this.props.error || this.props.isError || null}
             checked={this.props.checked}
-            disabled={this.props.disabled}
-            name={this.props.name}
-            onBlur={this.props.onBlur}
-            onFocus={this.props.onFocus}
+            // disabled={this.props.disabled}
```

Comment:
Should we rather just delete instead of commenting?

Reply:
Sorry for not being clear on this diff, I'm pushing here so that I can test on my windows computer. Attempting to remove all of the extra stuff from the elements to isolate the issue